addons/account/models/account_invoice.py

```diff
@@ -99,8 +99,12 @@ def _compute_residual(self):
         digits_rounding_precision = self.currency_id.rounding
         if float_is_zero(self.residual, precision_rounding=digits_rounding_precision):
             self.reconciled = True
+            if self.state == 'open':
```

Comment:
This make the module crash. In a compute field, it calls a new write and make strange things. I bet it is not the best location to put that code.